scripts/add_deprecations.sh

```diff
@@ -1,4 +1,4 @@
-#! /bin/bash
+#! /usr/bin/env bash
```

Comment:
I think this is a good change, but I wonder whether you were meaning to include it in this PR.

Reply:
I added the script and, if I had remembered, I would have used what Johan placed here: happy to include it in this PR!